### PR TITLE
remove angular-mocks dependency from bower.json

### PR DESCRIPTION
**Pull request by polomarcus**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -25,7 +24,6 @@
     "bower_components"
   ],
   "resolutions": {
-    "angular": "~1.3.13",
```
Comment (KostyaSha) [inline bower.json]: probably revert to the same value that was in 1.2.1

**Comment by polomarcus** [comment]
Hi @siddii,

When you will have the time, could you update the angular-timer bower package with this PR merged?

**Comment by siddii** [comment]
@polomarcus - I pushed a new version v1.3.1. Can you please verify if all looks OK?

**Comment by polomarcus** [comment]
Everything looks fine to me, thanks for merging the code @siddii

**Pull request merged**